Source/WebCore/platform/libwebrtc/LibWebRTCVPXVideoDecoder.cpp

```diff
@@ -169,7 +171,9 @@ LibWebRTCVPXInternalVideoDecoder::LibWebRTCVPXInternalVideoDecoder(LibWebRTCVPXV
     : m_outputCallback(WTFMove(outputCallback))
     , m_internalDecoder(createInternalDecoder(type))
     , m_useIOSurface(config.pixelBuffer == VideoDecoder::HardwareBuffer::Yes)
+    , m_isFullRange((type == LibWebRTCVPXVideoDecoder::Type::VP8 || type == LibWebRTCVPXVideoDecoder::Type::VP9 || type == LibWebRTCVPXVideoDecoder::Type::VP9_P2) && config.description.size() >= 12 && (*(config.description.data() + 6) & 0x1))
```

Comment:
It's deeply weird to add parsing of the VPx header here. What if you added the following to `VideoDecoder::Config`:

```c++
std::optional<bool> isFullRange { };
```

And passed in that information as part of the `VideoDecoder::create()` method?

Reply:
This is not part of this commit, but a stack of a now obsolete one which you commented earlier on and got addressed.